package.json

```diff
@@ -14,6 +14,7 @@
     "test:e2e": "cypress run",
     "test:e2e-chrome": "cypress run --browser chrome",
     "test:e2e-dev-server": "start-server-and-test start http://localhost:8080 test:e2e",
+    "test:e2e-interactive": "cypress open",
```

Comment:
This is why the `cypress` script exists in package.json. You can run 

    $ npm run cypress open

Reply:
Did you add this script because your IDE doesn't allow adding parameters to commands easily so it wouldn't be convenient for you to run the above command?

Reply:
I added this to be consistent with what we we are doing in the other lines above, e.g. `"test:e2e": "cypress run"`:
We are wrapping a cypress sub-command to get a couple of benefits:
* We describe what a script does using its name: IMHO `"test:e2e-interactive` speaks more to a developer than `cypress open`, I don't have to look up the docs what the specific subcommand dpes
* We can change the implementation of a specific script later on and keep its interface (the script name) stable
* (last and least): Yep, in my IDE I can open a little window showing all available npm scripts and pick a script only using keyboard strokes, which always saves a little time.

Reply:
Then let's keep both entries, this one and the `cypress` script. That let's us support both cli and ide workflows.